pom.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0"?>
+<!-- ~ Copyright (c) 2011-2014 The original author or authors ~ ~ All rights 
+  reserved. This program and the accompanying materials ~ are made available 
+  under the terms of the Eclipse Public License v1.0 ~ and Apache License v2.0 
+  which accompanies this distribution. ~ ~ The Eclipse Public License is available 
+  at ~ http://www.eclipse.org/legal/epl-v10.html ~ ~ The Apache License v2.0 
+  is available at ~ http://www.opensource.org/licenses/apache2.0.php ~ ~ You 
+  may elect to redistribute this code under either of these licenses. -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.vertx</groupId>
+    <artifactId>vertx-ext-parent</artifactId>
+    <version>6</version>
+  </parent>
+
+  <artifactId>vertx-mail</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <!--<name>Vert.x SMTP Service</name> -->
+
+  <properties>
+    <stack.version>3.0.0-SNAPSHOT</stack.version>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.vertx</groupId>
+        <artifactId>vertx-dependencies</artifactId>
+        <version>${stack.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-service-proxy</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-codegen</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-docgen</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-codetrans</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-lang-groovy</artifactId>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-lang-js</artifactId>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
```

Comment:
apache commons codec is used for Base64, Hexencoding etc, it should be possible to get rid of that and use the classes from vert.x instead
bouncy castle is used for the digest-md5 authentication, i would prefer not to implement that as an internal function, since crypto is always a bit tricky, not sure if jdk has an implementation of that
subetha is a test dependency, that shouldn't end up in the jar
commons email depends on javax.mail resp com.sun.mail, that basically does all the work, it would be possible to remove the use of commons email, but the api is really simpler than the one from sun (e.g. you do not have to fiddle with properties for the missing parameters, that is all encapsulated in the apache commons classes)